packages/stash/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "@latticexyz/stash",
```

```suggestion
  "private": true,
  "name": "@latticexyz/stash",
```
do we wanna make this private for now so we don't publish at the same time as explorer?